run/GCClassic/createRunDir.sh

```diff
@@ -389,9 +390,22 @@ if [[ ${met} = "ModelE2.1" ]]; then
 else
     RUNDIR_VARS+="RUNDIR_GCAP2_SCENARIO='not_used'\n"
     RUNDIR_VARS+="RUNDIR_GCAP2_RUNID='not_used'\n"
-    RUNDIR_VARS+="RUNDIR_VOLC_YEAR='\$YYYY'\n"
     RUNDIR_VARS+="RUNDIR_MET_AVAIL='# 1980-2021'\n"
-fi
+    
+    # Define the volcano paths for the HEMCO_Config.rc file
+    # NOTE: Benchmark simulations always use the climatological emissions!
+    if [[ "x${sim_name}" == "xfullchem" ]]  ||  \
```

Comment:
why are we prefixing these string comparisons with`x`?

Reply:
It's a bash idiom.  A good explanation is here: https://stackoverflow.com/questions/174119/why-do-shell-script-comparisons-often-use-xvar-xyes

I like using this because in case the `${sim_name}` is blank then using `[[ "x${sim_name}" == "xfullchem" ]]`ensures that you are comparing a string to a string.

Reply:
Ah okay that makes sense. Thanks!